README.md

```diff
@@ -84,50 +92,44 @@ fp login
 
 You will be then prompted to login with your account. When you complete the login you can safely close the window.
 
-### Step 2: Register a Proxy with Fiberplane
+### Step 2: Register an `fpd` (Fiberplane Daemon) with Fiberplane
 
-In order for the Proxy to receive queries from Fiberplane Notebooks, it needs to be authorized. This step will generate a **Proxy API Token** that will be needed in later steps.
+In order for `fpd` to receive queries from Fiberplane Notebooks, it needs to be authorized. This step will generate a **Daemon API Token** that will be needed in later steps.
 
-To register a proxy run a command `fp proxies create`:
+To register an `fpd` run a command `fp daemon create`:
 
 ```
-$ fp proxies create
+$ fp daemon create
 Added proxy "robust-antelope" # generates a random name
 Proxy API Token: XXX_XX # and a token - save this for later!
```

Comment:
Did we change the output of `fp` to say `Daemon API Token` as well?

Reply:
oh missed that one! It now looks like this

![CleanShot 2023-03-10 at 10 55 40@2x](https://user-images.githubusercontent.com/17225208/224285556-ad129b8c-4812-4804-ba24-b3ca509cd2fb.png)